File: beecrowd/tentados/1029-quantasChamadas.py
def fibonacci(a, lista):
    lista[0] +=1
    if (a == 0):
        return 0
    elif(a==1):
        return 1
    else:
        return fibonacci(a-1, lista)+fibonacci(a-2, lista)

def main():
    qtd = int(input())
    for i in range(qtd):
        chamadas=[0];
        num = int(input())
        fib = fibonacci(num, chamadas);
        chamadas[0]-=1
        print(f"fib({num}) = {chamadas[0]} calls = {fib}") 
    
main()

File: beecrowd/resolvidos/1029-quantasChamadas.cpp
#include <bits/stdc++.h>
using namespace std;

int fibonacci(int a, int *count){
    (*count)++;
    if (a == 0) return 0;
    else if(a==1) return 1;

    else {
        return fibonacci(a-1, count)+fibonacci(a-2, count);
    }
}

int main(void){
    int qtd, n, chamadas, fib;
    cin >> qtd;
    while(qtd--){
        cin >> n;
        chamadas=0;
        fib = fibonacci(n, &chamadas);
        cout << "fib(" << n << ") = " << chamadas-1 << " calls = " << fib << endl; 
    }
    return 0;
}